tests/src/test/scala/com/nvidia/spark/rapids/StringFallbackSuite.scala

```diff
@@ -25,12 +25,14 @@ class StringFallbackSuite extends SparkQueryCompareTestSuite {
     frame => frame.selectExpr("regexp_replace(strings,'a',strings)")
   }
 
-  testGpuFallback("String regexp_replace null cpu fall back",
-    "RegExpReplace",
-    nullableStringsFromCsv, execsAllowedNonGpu = Seq("ProjectExec", "Alias",
-      "RegExpReplace", "AttributeReference", "Literal")) {
-    frame => frame.selectExpr("regexp_replace(strings,null,'D')")
-  }
+  // This test is no longer valid since SPARK-28481 was implemented in 3.1 because
+  // the expression gets replaced with a null literal
+//  testGpuFallback("String regexp_replace null cpu fall back",
```

Comment:
How do we want to handle tests that are only valid for a subset of Spark versions that we support?

Reply:
this is likely the same as https://github.com/NVIDIA/spark-rapids/issues/382

Reply:
we likely need to add a way to handle different versions